cfme/automate/dialogs/__init__.py

```diff
@@ -54,8 +55,8 @@ class EditDialogView(DialogForm):
     @property
     def is_displayed(self):
         return (
-            self.in_customization and self.service_dialogs.is_opened and
-            self.label.text == self.label
+            self.in_customization and self.title.text == "Automate Customization" and
+            self.sub_title.text == "General" and self.label.text == self.label
```

Comment:
I don't think this is a good check - `self.text.label` won't equal `self.label` - `self.label` is the widget object, if the label (name) cannot be modified, you can validate `self.text.label` against `self.context['object'].label`.

Alternatively, if this is a weak check (edit page), return False here and update any navigation to this view to pass `wait_for_view=0`